app/models/exam_version.rb

```diff
@@ -45,4 +46,8 @@ def default_answers
       scratch: ''
     }
   end
+
+  def any_finalized?
```

Comment:
I think we're going to want an `any_started?` method to go along with this one...